Tests/SonarScanner.MSBuild.PreProcessor.Test/WebClientDownloaderTest.cs

```diff
@@ -34,26 +35,27 @@ public class WebClientDownloaderTest
         public void Ctor_SecurityProtocolIsDefault_RemainsDefault()
         {
             // Arrange
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.SystemDefault;
+            var securityProtocolHandlerMock = new Mock<ISecurityProtocolHandler>();
+            securityProtocolHandlerMock.Setup(x => x.SecurityProtocol).Returns(SecurityProtocolType.SystemDefault);
 
-            // Act
-            _ = new WebClientDownloader(null, null, new TestLogger());
+            // Act & Assert
+            _ = new WebClientDownloader(null, null, new TestLogger(), securityProtocolHandlerMock.Object, null, null);
 
-            // Assert
-            ServicePointManager.SecurityProtocol.Should().Be(SecurityProtocolType.SystemDefault);
+            securityProtocolHandlerMock.VerifySet(x => x.SecurityProtocol = SecurityProtocolType.Tls13 | SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls, Times.Never);
```

Comment:
you should verify with `Any()` that it's called `Never`

because it may be called with other arguments but you want to check it's never called with any argument, right?